komodo/release_transpiler.py

```diff
@@ -43,28 +71,68 @@ def _build(packages, py_ver, rhel_ver):
             version = versions
 
         if version:
-            release_dict[p] = version
+            release_dict[pkg_name] = version
     return release_dict
 
 
-def transpile_releases(matrix_file, output_folder):
+def _check_consistency_in_versions(
+    versions: OrderedDict, pkg_name: str, rhel_ver: str, py_ver: str
```

Comment:
Does this definitely need to be of type `OrderedDict`? It seems like `dict` should suffice. I think the object is coming from `load_yaml`, which I think uses `ruamel`, but maybe just check its type, and/or see if you can 'get away' with just a dict (which will be ordered anyway).

Reply:
Yes, indeed `OrderedDict` could be substituted overall. Thanks!